docs/cloud-edge/modules/_http_header_templates.mdx

```diff
@@ -71,9 +55,7 @@ Note: These values are only available when using the Compression module.
 | `${.conn.server_port}` | The port that the connection for this request came in on.                                       |
 | `${.conn.start_ts}`    | The timestamp when the TCP connection to the ngrok edge is established.                         |
 
-### Geo IP Template Variables {#ngrok-geo}
-
-This data is provided by [MaxMind GeoIP](https://www.maxmind.com/en/home).
```

Comment:
any specific reason to remove this line? i know we've been asked about where this data comes from in the past.